services/src/floodlight-modules/src/main/java/org/openkilda/floodlight/KildaCore.java

```diff
@@ -0,0 +1,78 @@
+/* Copyright 2018 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.floodlight;
+
+import org.openkilda.floodlight.config.provider.ConfigurationProvider;
+import org.openkilda.floodlight.service.CommandProcessorService;
+import org.openkilda.floodlight.service.of.InputService;
+import org.openkilda.floodlight.utils.CommandContextFactory;
+
+import com.google.common.collect.ImmutableList;
+import net.floodlightcontroller.core.IFloodlightProviderService;
+import net.floodlightcontroller.core.module.FloodlightModuleContext;
+import net.floodlightcontroller.core.module.FloodlightModuleException;
+import net.floodlightcontroller.core.module.IFloodlightModule;
+import net.floodlightcontroller.core.module.IFloodlightService;
+import net.floodlightcontroller.threadpool.IThreadPoolService;
+
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+
+public class KildaCore implements IFloodlightModule {
```

Comment:
Would be great to describe (add some docs?) the purpose of this class and perhaps modify classname to something like KildaCoreModule, because it is not really obvious for the first look.